sbt-cross/src/main/scala/sbtcross/CrossType.scala

```diff
@@ -0,0 +1,77 @@
+package sbtcross
+
+import sbt._
+
+import java.io.File
+
+abstract class CrossType {
+
+  /** The base directory for a (true sbt) Project
+   *  @param crossBase The base directory of the CrossProject
+   *  @param platform
+   */
+  def projectDir(crossBase: File, platform: Platform): File
+
+  final def platformDir(crossBase: File, platform: Platform): File =
+    projectDir(crossBase, platform)
```

Comment:
This is a simple alias to `projectDir`, which is useless. Choose one and discard the other.